tests/integration/test_ourlogs.py

```diff
@@ -174,32 +164,66 @@ def test_ourlog_extraction_with_sentry_logs(
         "body": "Example log record",
         "span_id": "eee19b7ec3c1b174",
         "severity_text": "info",
-        "severity_number": 10,
+        "severity_number": 9,
         "attributes": {
             "boolean.attribute": {"bool_value": True},
             "sentry.severity_number": {"int_value": 9},
             "sentry.severity_text": {"string_value": "info"},
         },
     }
-    expected_with_missing_fields = {
+
+    del ourlogs[0]["received"]
```

Comment:
We have a test helper which can fuzzy assert the time, `time_within{_delta}`: https://github.com/getsentry/relay/blob/bb8969728b9d3398138b7c5acd25d2da73a22f87/tests/integration/test_profile_chunks.py#L137-L149

Then you don't have to delete the time.